pyproject.toml

```diff
@@ -21,3 +21,4 @@ exclude = '''
 testpaths = ["tests", "superdesk", "apps", "content_api"]
 python_files = "*_test.py *_tests.py test_*.py tests_*.py tests.py test.py"
 asyncio_mode = "auto"
+asyncio_default_fixture_loop_scope = "function"
```

Comment:
Interesting discussion about this setting https://github.com/pytest-dev/pytest-asyncio/issues/924

Reply:
Yeah I read that, seems silly to me!